client.go

```diff
@@ -24,50 +25,59 @@ type Client struct {
 // Dial connects to an AMQP server.
 //
 // If the addr includes a scheme, it must be "amqp" or "amqps".
-// TLS will be negotiated when the scheme is "amqps".
-//
-// If no port is provided, 5672 will be used.
+// If no port is provided, 5672 will be used for "amqp" and 5671 for "amqps".
 func Dial(addr string, opts ...ConnOption) (*Client, error) {
 	u, err := url.Parse(addr)
 	if err != nil {
 		return nil, err
 	}
+	if u.Scheme == "" {
+		u.Scheme = "amqp"
+	}
```

Comment:
Can you remove this? It shouldn't be necessary anymore.

Reply:
Forgot to do this in the last commit. Removed.